ocs/workload.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import importlib
+import concurrent.futures
+
+
+log = logging.getLogger(__name__)
+
+
+class WorkLoad(object):
+    def __init__(
+        self, name=None, path=None, work_load=None, storage_type='fs',
+        pod=None
+    ):
+        """
+        Args:
+            name (str): name for this workload instance (for identifying in a
+                test run)
+            path (str): Mount point OR blk device on the pod where workload
+                should do IO (note: this need not be known at this
+                point in time)
+            work_load (str): example fio, mongodb, pgsql etc.
+            storage_type (str): type on which we will be running IOs,
+                if type is 'fs' we will interpret 'path' as mount point else
+                if type is 'block' we will interpret 'path' as a block device
+            pod (Pod): pod on which we want to run this workload
+        """
+        self.name = name
+        self.path = path
+        self.work_load = work_load
+        self.storage_type = storage_type
+        self.pod = pod
+
+        try:
+            self.work_load_mod = importlib.import_module(
+                f'workloads.{self.work_load}.{self.work_load}'
```

Comment:
Nice,

It took me some time to understand how you actually using fio.py below.
Could you please add a comment here what you are doing?

Reply:
Done, added comments